tests/conftest.py

```diff
@@ -6,7 +6,7 @@
 from importlib import import_module
 from sys import version_info as _version_info
 from types import ModuleType
-from typing import Callable, Type
+from typing import Any, Callable, Type
```

Comment:
Avoid using `Any` and follow other examples of signatures in this module.

Reply:
Removed